docs/how-to/integrate_oai_ran_with_usrp.md

```diff
@@ -0,0 +1,100 @@
+# Integrate Charmed OAI RAN with an Ettus Research USRP Radio
+
+This guide explains how to integrate Charmed OAI RAN with an Ettus Research USRP radio.
+
+The OAI RAN DU charm expects the USRP radio to be connected to the host machine via USB. 
```

Comment:
```suggestion
The OAI RAN DU charm expects the USRP radio to be connected to the host machine via USB 3. 
```

It will just not work if it is not USB 3.